### PR TITLE
Add select for Countries and states

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

File: src/utils/geo-data.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	}
+}
+
+/**
+ * Return an object from the global namespace or empty object if does not exist
+ *
+ * @param name
+ * @returns {*|{}}
+ */
+export function getObject( name ) {
```
Comment (bordoni) [inline src/utils/geo-data.js]: We should maybe move this into a more global Utils file, it's not specific to `geo-data` right?

File: src/utils/geo-data.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+/**
+ * Get name from a container object based on the code of the element otherwise an empty string.
+ *
+ * @param code
+ * @param container
+ * @returns {*|string}
+ */
+export function getName( code, container ) {
```
Comment (bordoni) [inline src/utils/geo-data.js]: Should we use the lodash `get` instead of this?

**Comment by mitogh** [comment]
Latest changes applied @bordoni 🎉

**Comment by bordoni** [comment]
Add the changelog and merge.

**Pull request merged**